Jenkinsfile.nightly

```diff
@@ -115,7 +115,7 @@ timestamps {
                   [$class: 'FileBinding', credentialsId:'OPS_SSH_KEY', variable: 'SSH_KEY'],
                   [$class: 'FileBinding', credentialsId:'OPS_SSH_PUB', variable: 'SSH_PUB'],
                   ]) {
-                    sh 'make -C e robotest-run-nightly'
+                    sh 'make -C e robotest-run ROBOTEST_CONFIG=nightly'
```

Comment:
This file will be removed in a later PR.  This change allows for a smooth cutover -- the current jenkins nightly config will continue working until I can update it -- probably the day after this merges.